ipinfo/cmd_tool.go

```diff
@@ -62,6 +66,10 @@ func cmdTool() error {
 	switch {
 	case cmd == "aggregate":
 		err = cmdToolAggregate()
+	case cmd == "is_v4":
+		err = cmdToolIs_v4()
+	case cmd == "is_v6":
+		err = cmdToolIs_v6()
```

Comment:
Func name should be `cmdToolIsV6` (Similar for v4), even though the actual subcmd has an underscore.